backend/api/invites.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any, Dict, Tuple, cast
+
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from backend.database_handler_entity import mongo
+
+invite_api = Blueprint('invite_api', __name__)
+
+
+@invite_api.route('/invite', methods=["GET"])
+@jwt_required()
+def invite() -> Tuple[Any, int]:
+    user_id = ObjectId(get_jwt_identity())
+
+    if (user_invite := mongo.select_invite(user_id)) is None:
+        return jsonify({}), 204
+    else:
+        document_id = str(user_invite["invite_document"])
+        document = cast(Dict, mongo.find_document(document_id))
+        return (
+            jsonify(
+                {
+                    "document_name": document["document_name"],
+                    "document_id": str(document["_id"]),
+                    "invite_id": str(user_invite["_id"]),
+                }
+            ),
+            200,
+        )
+
+
+@invite_api.route('/invite', methods=["POST"])
```

Comment:
Unite `GET /invite` and `POST /invite` to one function and use if-elif-else construction and separate functions to each HTTP method